src/main/java/org/aion/wallet/hardware/ledger/LedgerWallet.java

```diff
@@ -24,106 +30,73 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
+import java.util.concurrent.locks.ReentrantLock;
 import java.util.function.Consumer;
 import java.util.stream.Collectors;
 
 public class LedgerWallet implements HardwareWallet {
 
     private static final Logger log = WalletLoggerFactory.getLogger(LogEnum.WLT.name());
-    private static final String USER_DIR = System.getProperty("user.dir");
-    private static final String PATH = "Path";
-
-    private static final String RESPONSE = "response";
-    private static final String SPACE = " ";
-
-    private static final String NPM_COMMAND = "npm";
-    private static final String RUN_CMD = "run";
-    private static final String NPM_AION_HID_KEY = "get:aion-hid";
-
-    private static final String WINDOWS_DRIVER_PATH = USER_DIR + File.separator + "native\\win\\ledger\\Aion-HID\\npm.cmd";
-    private static final String WINDOWS_DRIVER_PATH_HID = USER_DIR + File.separator + "native\\win\\ledger\\Aion-HID\\hid";
-    private static final String WINDOWS_NPM_LOCATION = USER_DIR + File.separator + "native\\win\\ledger\\Aion-HID";
-    private static final String WINDOWS_PREFIX_KEY = "--prefix";
-
-    private static final String MAC_DRIVER_LOCATION = USER_DIR + File.separator + "native/mac/ledger/hid";
-    private static final String MAC_NPM_LOCATION = MAC_DRIVER_LOCATION + File.separator + "node-v8.11.4-darwin-x64/bin/";
-
-    private static final String LINUX_DRIVER_PATH = "native/linux/ledger/Aion-HID";
-    private static final String LINUX_PARAM_KEY = "--param=";
-
-    private static final int FIRST_INDEX = 0;
-    private static final byte PATH_LENGTH = 0x15;
-    private static final String AION_LEDGER_APP_PFX = "e0";
-    private static final String GET_PUBLIC_KEY_INDEX = "02";
-    private static final String SIGN_TRANSACTION_INDEX = "04";
-    private static final String DEFAULT_THIRD_AND_FORTH_BYTES_FOR_KEY_GENERATION = "0100";
-    private static final String DEFAULT_THIRD_AND_FORTH_BYTES_FOR_SIGNING = "0000";
-    private static final String DEFAULT_DERIVATION_PATH = "058000002c800001a98000000080000000";
-
-    private static final int HEX_KEY_SIZE = 64;
-
-    private static final WindowsNpmInstaller WINDOWS_NPM_INSTALLER = new WindowsNpmInstaller();
-    private static final String PREFIX = "--prefix";
 
     private static final int RETRIES = 3;
```

Comment:
We should remove the retry mechanism as well. We used to do this because the JS calls would sometimes fail if called in quick succession